Assets/Scripts/AI/AiPlayer.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+using UnityEngine;
+
+public class AiPlayer : AiActor
+{
+    //public ThirdPersonCharacter m_ThirdPersonCharacter;
+    public void Update()
+    {
+        //m_ThirdPersonCharacter.Move(m_Agent.desiredVelocity, false, false);
+    }
+}
```

Comment:
A script that has been completely commented out should not be pushed to the repo. Unless this is code removal, then perhaps remove the entire script.

Reply:
Okay will remove until completed